### ISSUE TITLE
Polygon -  News:Destiny 2 going down for maintenance before Faction Rally returns

### ISSUE THREAD

**Issue by sudocurse** [sudocurse/reading-list#3488]
Destiny 2 going down for maintenance before Faction Rally returns<br>
http://ift.tt/2DzxFKk<br>
  
  <img src="http://ift.tt/2D5obVX">

  <p>Another hotfix incoming</p> <p id="Z0rvzo"><a href="http://ift.tt/2tW3hEc"><em>Destiny 2</em></a>’s servers will go offline tomorrow, Jan. 16, so Bungie can conduct maintenance and push a new patch live.</p>
<p id="RlSPR4">The <a href="http://ift.tt/1fqcdKQ">maintenance window</a> will begin at 11 a.m. ET, at which point people who aren’t already playing will no longer be able to log in to the game’s servers. An hour later, at 12 p.m. ET, all players will be kicked to the title screen. Bungie expects to finish its work at 3 p.m. ET.</p>
<p id="s2PO1N">While the <em>Destiny 2</em> servers are undergoing maintenance, Bungie will issue a hotfix that will update the game to version 1.1.1.3. Right now, players must own the <a href="http://ift.tt/2p93sOE"><em>Curse of Osiris</em></a> expansion to access <a href="http://ift.tt/2zzG8tL">seasonal content</a> such as Faction Rallies and Iron Banner; this patch will fully eliminate that requirement, making those events free for all <em>Destiny 2</em> players.</p>
<div class="c-float-right"><aside id="JRaXvh"><div data-anthem-component="readmore" data-anthem-component-data='{"stories":[{"title":"Bungie says it’s scaling back Eververse in Destiny 2 content roadmap","url":"http://ift.tt/2DlaQtR"}]}'></div></aside></div>
<p id="C6gL9o">Once maintenance is complete and the game’s servers come back online, the first Faction Rally of 2018 will begin. Running from Jan. 16-23, this event will feature new armor ornaments in addition to the standard gear. On the weapon front, each of the three factions is offering an auto rifle this time around. As always, the gun from the winning group will be available for purchase at a steep discount to that faction’s followers — and an exorbitant sum for everyone else. This is the Faction Rally that Bungie <a href="http://ift.tt/2nSvXzu">postponed back in December</a>.</p>
<p id="Zppa5V">Making seasonal content free for all <em>Destiny 2</em> owners is just one part of Bungie’s efforts to rehabilitate the game and win back players’ hearts and minds. The studio announced a <a href="http://ift.tt/2DlaQtR">suite of upcoming changes</a> in a lengthy post last week, saying it would reduce the prevalence of the Eververse microtransaction store and deliver new content in free updates over the coming months.</p>

<br><br>
via Polygon -  News http://ift.tt/2po1Ka8<br>
January 15, 2018 at 12:15PM
